lib/stdlib/src/sofs.erl

```diff
@@ -440,7 +576,35 @@ to_external(S) when ?IS_SET(S) ->
 to_external(S) when ?IS_ORDSET(S) ->
     ?ORDDATA(S).
 
--doc "Returns the [type](`m:sofs#type`) of an atomic, ordered, or unordered set.".
+-doc """
+Returns the [type](`m:sofs#type`) of an atomic, ordered, or unordered set.
+
+## Examples
+
+```erlang
```

Comment:
Should we break this into three separate code blocks with the comment being places inbetween? similar to how is done for `specification/1`?

Reply:
Yes, will do.